components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/actuate/endpoint/CamelRoutesEndpoint.java

```diff
@@ -76,6 +74,11 @@ public void startRoute(String id) throws Exception {
         camelContext.getRouteController().startRoute(id);
     }
 
+    public void resetRoute(String id) throws Exception {
+        ManagedRouteMBean resetRoute = camelContext.getManagedRoute(id, ManagedRouteMBean.class);
+        resetRoute.reset();
```

Comment:
Yeah I am torn between 204 and 404. Either its just OK but with a different status code. Or its a client error with 4xx. Maybe a 404 is more correct? The route id do not exists and therefore the url is not found. But since 404 is also used for urls where you type a totally wrong like /camel/doesnotexistsanditypesomecrapehere. @lburgazzoli @zregvart @nicolaferraro @oscerd @gzurowski  any thoughts? And can we look at what other spring boot actuators do in this situation?